### ISSUE TITLE
TypeError: Cannot call method 'resume' of undefined

### ISSUE THREAD

**Issue by jfrux** [tellnes/node-apacheconf#2]
/Users/joshua/Project/node_modules/apacheconf/index.js:227
  return this._getProp('_stream').resume()
                                  ^
TypeError: Cannot call method 'resume' of undefined

**Comment by jfrux** [comment]
If I comment out the .resume() line it seems to go away, but my understanding of this module is slim at this moment.

**Comment by tellnes** [comment]
Do you have a reproducible test case?

**Comment by jfrux** [comment]
Only similar issue I could find via google is https://github.com/SublimeLinter/SublimeLinter/issues/154
And it doesn't appear to be resolved...

**Comment by tellnes** [comment]
It doesn't seems relevant.

Your test case works fine for me. What version of node are you using?
Also, the module loads `Include` statements, so it could be in the file `/usr/local/etc/apache2/extra/proxy-html.conf` which is included at line 495.

Also, I could try to add some debug statements tomorrow. Then you could send me the log output.

**Comment by jfrux** [comment]
Node v0.10.12
Okay, it's definitely the Included file causing it.
However, the included file is a default file never altered.
Same goes for the httpd.conf.

These conf files are from Apache 2.4 also, just fyi.

I'm going to continue testing.

Thanks!

**Comment by tellnes** [comment]
Yes, I expect it is the default files, but they is not default on my system (Ubuntu).

I've added some debug statmests. Do you mind trying out the master branch? To see the debug output, set the `DEBUG` enviroment variable to `apacheconf`. Eg. `DEBUG=apacheconf node example.js`.
